monitoring/dashboard.json

```diff
@@ -0,0 +1,4553 @@
+apiVersion: v1
```

Comment:
This is configMap YAML which should have .yml /. yaml extension.
We can keep the JSON as a separate file and load in using helm templating

```
data:
  {{-/* Note, indent of only two spaces */}}
  config.json: |-
{{ .Files.Get "my-config.json" | indent 4 }}
{{/* .Get, not .Glob; indent 4 spaces, more than 2 above */}}
```

Reply:
Updated the file extension from dashboard.json to dashboard.yaml